base_exception/hooks.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion, Mourad EL HADJ MIMOUNE
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+def pre_init_hook(cr):
+    """If coming from v9 where only `sale_exception` is installed, when
+    auto-installing this one by dependencies, we should take care of
+    renaming previous table before the new model is loaded.
+    """
+    cr.execute("SELECT 1 FROM pg_class WHERE relname = 'sale_exception'")
+    if not cr.fetchone():
+        return  # fresh install, so exiting
+    from openupgradelib import openupgrade
```

Comment:
As we are not in the context of openupgrade,  penupgradelib is an external lib. Should not be nice to add and external dependency in manifest ? And manage import exception like: 
try:
   from openupgradelib import openupgrade
except ImportError:
    ....
?

Reply:
Not needed. That's why I import after the checking of the column: if you are fresh installing the module, no need of this lib. But if you come from previous version, then you have already openupgradelib due to OpenUpgrade, so also not a problem.